main.js

```diff
@@ -29,9 +30,139 @@ function checkWindowSize() {
+      source_link: 'https://github.com/marvin-nyalik/DestryMarkers',
+    },
+    {
+      name: 'Walker',
+      description: 'how to keep walking when you cannot keep walking',
+      image: 'images/project_fourth.png',
+      technologies: ['HTML', 'CSS', 'JavaScript'],
+      live_link: 'https://marvin-nyalik.github.io/DestryMarkers/',
+      source_link: 'https://github.com/marvin-nyalik/DestryMarkers',
+    },
+  ];
+  const projectContainer = document.getElementById('works');
+  projectDetails.forEach((project, index) => {
+    const articleTemplate = `
+      <article class="card">
+        <img src="${project.image}" alt="project1" class="card-project">
+        <div>
+          <h2>${project.name}</h2>
+          <ul>
+            <li class="no-bullet">CANOPY</li>
+            <li class="project-tag">
+              <span>
+                <svg width="8" height="8" viewBox="0 0 8 8" fill="none" xmlns="http://www.w3.org/2000/svg">
+                  <circle cx="4" cy="4" r="4" fill="#C1C7D0"/>
+                </svg>
+                Back End Dev
+              </span>
+            </li>
+            <li class="project-tag">
+              <span>
+                <svg width="8" height="8" viewBox="0 0 8 8" fill="none" xmlns="http://www.w3.org/2000/svg">
+                  <circle cx="4" cy="4" r="4" fill="#C1C7D0"/>
+                </svg>
+                2015
+              </span>
+            </li>
+          </ul>
+          <p>${project.description}</p>
+          <ul class="tech-stack">
+            <li>${project.technologies[0]}</li>
+            <li>${project.technologies[1]}</li>
+            <li>${project.technologies[2]}</li>
+          </ul>
+          <button class="btn" type="button">See Project</button>
+        </div>
+        <div class="modal-cover" id="modal-cover">
+          <div class="modal" id="modal${index}">
+            <div class="modal-head-container">
+              <h2>${project.name}</h2>
+              <span class="modal-close">&times;</span>
+            </div>
+            <ul class='modal-ul'>
+              <li class="no-bullet">CANOPY</li>
+              <li class="project-tag">
+                <span>
+                  <svg width="8" height="8" viewBox="0 0 8 8" fill="none" xmlns="http://www.w3.org/2000/svg">
+                    <circle cx="4" cy="4" r="4" fill="#C1C7D0"/>
+                  </svg>
+                  Back End Dev
+                </span>
+              </li>
+              <li class="project-tag">
+                <span>
+                  <svg width="8" height="8" viewBox="0 0 8 8" fill="none" xmlns="http://www.w3.org/2000/svg">
+                    <circle cx="4" cy="4" r="4" fill="#C1C7D0"/>
+                  </svg>
+                  2015
+                </span>
+              </li>
+            </ul>
+            <img src="${project.image}" alt="project1" class="modal-img">
+            <div class="modal-footer">
```

Comment:
- Yoiur design for this highlighted section on the desktop does not completely match the provided figma template please kindly work on fixing it. :+1:  
- [ ] As seen from your design your button lacks the proper amount of spacing required
- [ ] The hover function for the button is also not correct as once it is hover on the ackground color turns purple and the text can no longer be seen as it becomes invisible please kindly provide a color for the text in white so that it becomes visible when the button is hovered on :+1: :sparkles: 

- figma design
![bb-f](https://github.com/marvin-nyalik/NyalikPortfolio/assets/89055502/1d9513c2-c1b3-436c-8ec1-e9a5aa96775b)
- your design
![bb-s](https://github.com/marvin-nyalik/NyalikPortfolio/assets/89055502/0bf63090-fa56-45c5-bfe9-141a55e12907)